tools/proofers/codepoint_validator.js

```diff
@@ -58,9 +62,67 @@ $(function () {
     });
 
     $("#cc-remove").click(function () {
-        let textarea = $("#text_data");
-        textarea.val(textarea.val().replace(pattern , ""));
+        textArea.value = textArea.value.replace(badPattern , "");
         $("#checker").hide();
         $("#proofdiv").show();
     });
+
+    var above = {
+        "=": "\u0304", // macron
+        ":": "\u0308", // diaeresis
+        ".": "\u0307", // dot
+        "`": "\u0300", // grave
+        "'": "\u0301", // acute
+        "^": "\u0302", // circumflex
+        ")": "\u0306", // breve
+        "~": "\u0303", // tilde
+        ",": "\u0327", // cedilla
```

Comment:
I think this should be `\u0309` for the "hook above" instead of cedilla again.

Reply:
This was because of the lower case g with cedilla which appears above the letter. I can change it or add another symbol for hook.